ibm/resource_ibm_schematics_workspace.go

```diff
@@ -0,0 +1,1514 @@
+							Optional:    true,
+							Description: "The entitlement key that you want to use to install IBM Cloud entitled software.",
+							Elem:        &schema.Schema{Type: schema.TypeMap},
+						},
+						"namespace": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The Kubernetes namespace or OpenShift project where the resources of all IBM Cloud catalog templates that are included in the catalog offering are deployed into.",
+						},
+						"region": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The IBM Cloud region that you want to use for the resources of all IBM Cloud catalog templates that are included in the catalog offering.",
+						},
+						"resource_group_id": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The ID of the resource group that you want to use for the resources of all IBM Cloud catalog templates that are included in the catalog offering.",
+						},
+						"worker_count": &schema.Schema{
+							Type:        schema.TypeInt,
+							Optional:    true,
+							Description: "Cluster worker count.",
+						},
+						"worker_machine_type": &schema.Schema{
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "Cluster worker type.",
+						},
+					},
+				},
+			},
+			"tags": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "A list of tags that are associated with the workspace.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"template_env_settings": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "A list of environment variables that you want to apply during the execution of a bash script or Terraform action. This field must be provided as a list of key-value pairs, for example, **TF_LOG=debug**. Each entry will be a map with one entry where `key is the environment variable name and value is value`. You can define environment variables for IBM Cloud catalog offerings that are provisioned by using a bash script.",
+				Elem:        &schema.Schema{Type: schema.TypeMap},
+			},
+			"template_git_folder": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The subfolder in your GitHub or GitLab repository where your Terraform template is stored.",
+			},
+			"template_init_state_file": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The content of an existing Terraform statefile that you want to import in to your workspace. To get the content of a Terraform statefile for a specific Terraform template in an existing workspace, run `ibmcloud terraform state pull --id <workspace_id> --template <template_id>`.",
+			},
+			"template_type": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
```

Comment:
your are saying it required and if not provided it uses terraform v0.11

Reply:
template_type field is a required field. This is a documentation defect from the Schematics API. Updated the documentation and raised a documentation defect internally